web-ui/src/api/api.js

```diff
@@ -1,4 +1,4 @@
-import axios from "axios";
+// import fetch from "node-fetch";
```

Comment:
Why is this commented out if 'fetch' is used below?

Reply:
It actually should be removed. We shouldn't be using node-fetch here, but rather the browser provided global`fetch`. I'll remove this commented line. Thanks!